virus/Orthocurvulavirus_phantasma.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Orthocurvulavirus phantasma"
+output:
+  html_document: default
+  pdf_document: default
+  
+---
+# Hide-and-seek, can you find a ghost?: Identification of novel dsRNA virus from the mushroom meta-data
+written by: [Matin Muradli](https://github.com/yourname *optional link*)
+
+### Orthocurvulavirus phantasma
+*Orthocurvulavirus is the genus of the top three closest hits in blastx. Phantasma means phantom, ghost or apparation in latin. I chose this name, because the suspected RdRp CDS was hidden inside of another ORF, which makes it harder to find. Thus, as a ghost, even if one looks into right direction, it is almost impossible to see it.*
+
+![Generative AI Cover-art](img/Orthocurvulavirus phantasma/cover.jpg)
```

Comment:
The Link pats you're using are not rendering, the files are stored in:
`virus/img/Orthocurvulavirus phantasma/Fig_1_RdRp.png`based on how you organized things. Fix htis for all figures.